cmd/adapter/main.go

```diff
@@ -33,6 +33,8 @@ type Adapter struct {
 }
 
 var logger = klogr.New().WithName("keda_metrics_adapter")
+var metricsPort int
+var metricsServer prommetrics.Server
```

Comment:
Minor nit. Might be better to avoid naming these vars `metricsServer` and `metricsPort`. This whole adapter is a metrics server (just a different kind) so to avoid confusion I'd rename it, eg. `prometheusServer` or `promMetricsServer`, etc.

Reply:
Totally agree. Good suggestion. I'll revise.